pkg/ovsdb/client/client.go

```diff
@@ -122,3 +123,78 @@ func NewNbClient(addr string) (client.Client, error) {
 
 	return c, nil
 }
+
+// NewSbClient creates a new OVN SB client
+func NewSbClient(addr string, insecureSkipVerify bool) (client.Client, error) {
```

Comment:
Can we merge `NewSbClient()` and `NewNbClient()` to resue code?

Reply:
i will try